Makefile

```diff
@@ -214,6 +214,25 @@ image-build-iso: rpm
 iso: image-build-configure image-build-iso
 .PHONY: iso
 
+RPM_BUILDER_IMAGE_TAG=$(shell \
+  gawk 'BEGIN { FS=":" } \
+	/FROM/ { \
+		match($$0, /rhel-([0-9]+).*golang-([0-9]\.[0-9]{2})-openshift-([0-9]+\.[0-9]+)/,m);\
+		printf "%s-rhel%s-go%s",m[3],m[1],m[2]; }' packaging/images/Containerfile.rpm-builder)
```

Comment:
Since we're building the container from make, can we simplify this by defining a variable in the Makefile and passiing it to the Containerfile?

```
Makefile:
GOLANG_CONTAINER=release:rhel-8-release-golang-1.19-openshift-4.13

podman build -v GOLANG_CONTAINER=$GOLANG_CONTAINER ...

Containerfile:
FROM registry.ci.openshift.org/openshift/$GOLANG_CONTAINER
```

Reply:
Following your suggestion, does `microshift-builder:rhel-8-release-golang-1.19-openshift-4.13` sound good as an image `name:tag`?

Reply:
I think so, because the CI image name/version is the main identifier for it.
Unless, of course, there's a limitation in docker not allowing to use that name as-is.